### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by flying-pi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,37 @@
 # rg_instructor_analytics
 
+## Install
```
Comment (idegtiarov) [inline README.md]: s/Install/Installation/

File: README.md [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
+* Add `rg_instructor_analytics` to installed app
```
Comment (idegtiarov) [inline README.md]: s/to installed/to the installed/ Please add the path to the settings file and move subsection down to the other subsection which relates settings file.

File: README.md [hunk 3]
```diff
@@ -1,18 +1,37 @@
 # rg_instructor_analytics
 
+## Install
+* Add `rg_instructor_analytics` to installed app 
+* Add to lms url:
```
Comment (idegtiarov) [inline README.md]: Please add the path to the URL file

File: README.md [hunk 4]
```diff
@@ -1,18 +1,37 @@
...
+```python
+url(
+        r'^courses/{}/tab/instructor_analytics/'.format(
+            settings.COURSE_ID_PATTERN,
+        ),
+        include('rg_instructor_analytics.urls'),
+        name='instructor_analytics_endpoint',
+    ),
+```
+* Add in to the settings file:
```
Comment (idegtiarov) [inline README.md]: ditto (path to settings file)

File: README.md [hunk 5]
```diff
@@ -1,18 +1,37 @@
...
+        ),
+        include('rg_instructor_analytics.urls'),
+        name='instructor_analytics_endpoint',
+    ),
+```
+* Add in to the settings file: 
+```python
+FEATURES['ENABLE_XBLOCK_VIEW_ENDPOINT'] = True
+```
+* Run:
```
Comment (idegtiarov) [inline README.md]: s/Run/Run in the console/

**Review by idegtiarov — CHANGES_REQUESTED** [review]
Some nits and comments are inline

File: README.md (hunk 2 above)
Comment (flying-pi) [reply README.md]: I think - incorrect to add some file for adding this because we can setup it in different places.

File: README.md (hunk 3 above)
Comment (flying-pi) [reply README.md]: It is commonly known information for a specialist, who install such things.

File: README.md (hunk 1 above)
Comment (flying-pi) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (flying-pi) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (idegtiarov) [reply README.md]: You write readme for all potential users of the application not only DevOps engineers. So please add the path.

File: README.md [hunk 6]
```diff
@@ -1,18 +1,37 @@
 # rg_instructor_analytics
 
+## Installation
+* Add `rg_instructor_analytics` to the installed app
```
Comment (idegtiarov) [inline README.md]: You write the readme file for all potential users of the application not only DevOps engineers. So please add the paths in all places which need to be updated.

File: README.md [hunk 7]
```diff
@@ -1,18 +1,37 @@
 # rg_instructor_analytics
 
+## Installation
+* Add `rg_instructor_analytics` to the installed app 
+* Add to lms url:
```
Comment (idegtiarov) [inline README.md]: ditto

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (flying-pi) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (flying-pi) [reply README.md]: done

File: README.md (hunk 7 above)
Comment (flying-pi) [reply README.md]: done

**Review by idegtiarov — APPROVED** [review]

**Pull request merged**